src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTag.java

```diff
@@ -6,30 +6,34 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
 /**
+ * Config:
+ * tag = &#64;author
+ * tagFormat = \\S
+ *
  * Testing tag writing
  * @author Daniel Grenner
  * @incomplete This class needs more code...
  * @doubletag first text
  * @doubletag second text
  * @emptytag
  */
-class InputWriteTag
+class InputWriteTag // violation at 14
```

Comment:
if violation occurred at 14, why comment is here? :)

Reply:
I didn't wanted to amend the violation message with ` // violation` at the end. Well changes are done now.

Reply:
Please resolve this one if everything is ok.